src/component/canvas/ScatterRenderable.jsx

```diff
@@ -111,6 +111,8 @@ export class ScatterRenderable extends React.PureComponent {
         index: number
     ): ?React.Element<any> {
         if(row.x == null || row.y == null) return null;
+        if(isNaN(row.x) || isNaN(row.y)) return null;
```

Comment:
I thought the same, but didn't want the line to be so long and looked at writing some kind of 'is number' function but also opted against that because I'd rather use something third party. 

How do you feel about using something like lodash's isNumeric()? It will cover both the cases the above, and the case you mentioned about the value not actually being a number